restricted_site_access.php

```diff
@@ -412,7 +412,7 @@ public static function restrict_access_check( $wp ) {
 				}
 				// Fall thru to case 3 if case 2 not handled.
 			case 3:
-				$message  = esc_html( self::$rsa_options['message'] );
+				$message  = wp_kses( self::$rsa_options['message'], self::get_allowed_message_tags() );
```

Comment:
@pattonwebz  can we use `wp_kses_post` here instead to simplify this PR?

Reply:
Swapped this here and in the other places referenced.